client/my-sites/customer-home/cards/tasks/site-setup-list/current-task-item.jsx

```diff
@@ -20,11 +20,15 @@ const CurrentTaskItem = ( { currentTask, skipTask, startTask, useAccordionLayout
 					</Badge>
 				) : (
 					<div className="site-setup-list__task-timing task__timing">
-						<Gridicon icon="time" size={ 18 } />
-						{ translate( '%d minute', '%d minutes', {
-							count: currentTask.timing,
-							args: [ currentTask.timing ],
-						} ) }
+						{ currentTask.timing && (
```

Comment:
The decision here is to make the `CurrentTaskItem` more generic to adhere with designs.